tests/core/vm/test_nonce_abstraction.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+
+from eth_utils import (
+    decode_hex,
+)
+
+from evm.utils.padding import pad32
+
+from tests.core.helpers import (
+    new_sharding_transaction,
+)
+
+
+DIR = os.path.dirname(__file__)
+nonce_tracking_contracts = json.load(
+    open(os.path.join(DIR, '../contract_fixtures/nonce_tracking_contracts.json'))
```

Comment:
Note to self:

This would make a good generic fixture.  Something that loads all of the json files from `contract_fixtures` and exposes them as a generic fixture so that they can be used across the entire test suite without any boilerplate.